common/libretro.c

```diff
@@ -273,11 +273,6 @@ int Sys_FileTime(const char *path)
    return buf.st_mtime;
 }
 
-void Sys_mkdir(const char *path)
```

Comment:
Why did you do this though? Now it doesn't compile anymore for several platforms, such as iOS and Windows.

iOS -
http://p.0bl.net/738

Windows -
http://p.0bl.net/739

Reply:
Oh wow, that was a really stupid mistake... please accept my apologies...

I'll make a new pull request with that function restored (I'm still very inexperienced with git, and I'm having to borrow someone else's PC to do this, so I hope the pull request comes through all right...)

Reply:
Alright.

Reply:
Okay, #60 should fix this...